frontend/stubs/list_methods.py

```diff
@@ -0,0 +1,52 @@
+"""Stub file for methods invoked on lists"""
+from typing import TypeVar, List
+
+T = TypeVar("T")
+
+
+def append(l: List[T], e: T) -> None:
```

Comment:
Okay. Could you add a test case that makes sure that using them without receivers doesn't work?